XenModel/Actions/VM/VMCrossPoolMigrateAction.cs

```diff
@@ -41,9 +40,10 @@ public class VMCrossPoolMigrateAction : PureAsyncAction
     {
         private readonly VmMapping mapping;
         private readonly XenAPI.Network transferNetwork;
+        private readonly bool copy;
 
-        public VMCrossPoolMigrateAction(VM vm, Host destinationHost, XenAPI.Network transferNetwork, VmMapping mapping)
-            : base(vm.Connection, GetTitle(vm, destinationHost))
+        public VMCrossPoolMigrateAction(VM vm, Host destinationHost, XenAPI.Network transferNetwork, VmMapping mapping, bool copy)

```

Comment:
Could we just comment it to make it clearer, rather than make an extra function or enum?